alibi_detect/cd/base.py

```diff
@@ -80,9 +80,6 @@ def __init__(
         if isinstance(train_size, float) and isinstance(n_folds, int):
             logger.warning('Both `n_folds` and `train_size` specified. By default `n_folds` is used.')
 
-        if preds_type not in ['probs', 'logits']:
-            raise ValueError("'preds_type' should be 'probs' or 'logits'")
-
```

Comment:
Needs to be moved to each backend specifically:
*  pytorch and tensorflow support probs and logits
* sklearn supports probs and scores

Reply:
Should the docstring be updated also?

Reply:
I don't think so ... why would that be the case?

Reply:
The base class can take `probs`, `logits` and `scores` now right?

Reply:
Yes, but I moved that check into each backend. So for `tensorflow` & `pytorch` would be `['probs', 'logits']` and for `sklearn` would be `['probs', 'scores']`. For each backend the docs are updated accordingly. The base class now is just storing the `preds_type`.

Reply:
OK but the base class docstring is probably slightly confusing still:
```
        preds_type
            Whether the model outputs probabilities or logits
```